### DISCUSSION TITLE
Windows Installation without Grabber

### DISCUSSION THREAD

**Discussion by n0valis** [awawa-dev/HyperHDR#53] [Q&A]
I run most of my HDR content from a Windows HTPC Kodi machine.
Is it possible to run HyperHDR on the PC and process its own screen without grabbing the output? 
Then send it via network to the ESP8266.
I use SK6812 RGBNW LEDs. Anything to take care of regarding the best firmware for the D1 mini and configurationwise in HyperHDR?

**Comment by awawa-dev** [comment]
No, there is no screen grabber in HyperHDR as it was always intended to isolate video player from the capturing (and leave it to the external usb grabber) because otherwise you may experience side effects that affect video quality/acceleration. As MS2109 costs below $10 you won't loose much if it doesn't meet your expectations ;)

**Reply by n0valis** [reply]
Would it be possible then to run HyperionScreenCap on the same machine?

**Reply by awawa-dev** [reply]
Yes, but without hdr tone mapping support for HyperionScreenCap https://github.com/awawa-dev/HyperHDR/issues/46
